src/registrar/management/commands/utility/extra_transition_domain_helper.py

```diff
@@ -198,6 +196,28 @@ def update_transition_domain_models(self):
                     f"{TerminalColors.ENDC}"
                 )
                 failed_transition_domains.append(domain_name)
+        
+        updated_fields = [
+            "organization_name", 
+            "organization_type", 
+            "federal_type", 
+            "federal_agency", 
+            "first_name", 
+            "middle_name", 
+            "last_name", 
+            "email", 
+            "phone", 
+            "epp_creation_date", 
+            "epp_expiration_date",
+        ]
+
+        batch_size = 1000
+        # Create a Paginator object. Bulk_update on the full dataset
+        # is too memory intensive for our current app config, so we can chunk this data instead.
+        paginator = Paginator(updated_transition_domains, batch_size)
+        for page_num in paginator.page_range:
+            page = paginator.page(page_num)
+            TransitionDomain.objects.bulk_update(page.object_list, updated_fields)
```

Comment:
Including this change in here since we often need to run the transfer_transition_domain script for datasets with these sorts of tickets. This just improves the load time of it. Not by a lot but it makes it less-so. Works the same as before.